lib/game/game.dart

```diff
@@ -0,0 +1,400 @@
+import 'dart:math' as math;
```

Comment:
this is a total breaking change, is it necessary?

Reply:
This is not a breaking change, in fact, like I said on the PR description, I did this, exactly to avoid breaking changes.

Reply:
Actually, I got confused by the files,  but still this is not a breaking change, look at the original lib/game.dart file

Reply:
gotcha, I was mislead by github as well